scattering/van_hove.py

```diff
@@ -15,13 +14,16 @@
 def compute_van_hove(
     trj,
     chunk_length,
-    parallel=False,
+    parallel=True,
+    chunk_starts=None,
+    cpu_count=None,
     water=False,
     r_range=(0, 1.0),
     bin_width=0.005,
     n_bins=None,
     self_correlation=True,
     periodic=True,
+    num_concurrent_paris=100000,
```

Comment:
`num_concurrent_pairs` has a spelling typo.  Could this also be changed to `n_concurrent_pairs`?